File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/com/github/asyncmc/mojang/sessions/scala/gatling/model/SessionApiError.scala

package com.github.asyncmc.mojang.sessions.scala.gatling.model


case class SessionApiError (
    /* A message describing the error */
    _error: Option[String],
    /* The request path */
    _path: Option[String]
)
object SessionApiError {
    def toStringBody(var_error: Object, var_path: Object) =
        s"""
        | {
        | "error":$var_error,"path":$var_path
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-sessions/src/main/kotlin/com/github/asyncmc/mojang/sessions/kotlin/spring/model/SessionApiError.kt
package com.github.asyncmc.mojang.sessions.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * An object describing giving a bit of information about the error
 * @param error A message describing the error
 * @param path The request path
 */
data class SessionApiError (

        @JsonProperty("error") val error: String? = null,

        @JsonProperty("path") val path: String? = null
) {

}

